### DISCUSSION TITLE
Long pause after each sentences. How do i fix this?

### DISCUSSION THREAD

**Discussion by HoodieMc** [jing332/tts-server-android#219] [Q&A]
I don't really understand what's going on... I just found this app from reddit link and im having issue using this as an engine for another Ereader app. TTS server itself is great but when it comes to the Ereader there's a long pause after every paragraph and sentences. I'm wondering how i can remove this delay.

**Comment by s3webmaster** [comment]
> I don't really understand what's going on... I just found this app from reddit link and im having issue using this as an engine for another Ereader app. TTS server itself is great but when it comes to the Ereader there's a long pause after every paragraph and sentences. I'm wondering how i can remove this delay.

I'm facing exactly the same issue!  Please can someone help?

**Reply by petulko72** [reply]
Hi, try this [fork](https://github.com/mobad/tts-server-android/releases/tag/1.24.042106):  - works without any pause....

**Reply by Sleepnlofi** [reply]
Made and account to say your a legend! Thanks 🙏
